docs/src/Makefile

```diff
@@ -153,3 +154,6 @@ doctest:
 	$(SPHINXBUILD) -b doctest $(ALLSPHINXOPTS) $(BUILDDIR)/doctest
 	@echo "Testing of doctests in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/doctest/output.txt."
+
+show:
```

Comment:
> As-is I think this fails if there is no build, e.g. "make clean show" _always_ fails. Whereas, AFAICT every other build-target is self-contained. So, I _think_ this fixes that.. (WFM)

Sorry, this doesn't work !
If you do that and say "make html-noplot show" then it builds twice.
I'm afraid it's too long since I did makefiles, and I'm not sure how to do this.

Reply:
Yes, I think you somehow need the subsidiary target to represent the **_outputs_** of the build process, not the process itself.  
Like, classically, you would say which complied sources are needed to build an executable with a linker command, and then another rule shows how to make those from sourcefiles with the compiler.
But I frankly don't know how to do it in this case.  
( I expect sphinx provides some special way of listing all relevant files of a given type ? )

Reply:
@pp-mo what behaviour do you see with `make clean show`?  I just got a firefox tab displaying "file not found", which seems like reasonable/acceptable behaviour to me.

Reply:
> @pp-mo what behaviour do you see with `make clean show`? I just got a Firefox tab displaying "file not found", which seems like reasonable/acceptable behaviour to me.

Well that is what it does for me, but I guess I was expecting a bit more intelligence from a makefile, because that is what makefiles are really for !
I guess it's no big deal, since it seems quite complicated to do what I'd hoped might be simple :cry:

Reply:
Thanks @pp-mo.  I have to admit I don't have any background on Makefiles in general, so I'm certainly missing some perspective here.